benchmarks/Microsoft.AspNetCore.SignalR.Microbenchmarks/BroadcastBenchmark.cs

```diff
@@ -40,13 +40,16 @@ public void GlobalSetup()
                 protocol = new MessagePackHubProtocol();
             }
 
+            var options = new PipeOptions();
             for (var i = 0; i < Connections; ++i)
             {
-                var pair = DuplexPipe.CreateConnectionPair(PipeOptions.Default, PipeOptions.Default);
+                var pair = DuplexPipe.CreateConnectionPair(options, options);
                 var connection = new DefaultConnectionContext(Guid.NewGuid().ToString(), pair.Application, pair.Transport);
                 var hubConnection = new HubConnectionContext(connection, Timeout.InfiniteTimeSpan, NullLoggerFactory.Instance);
                 hubConnection.Protocol = protocol;
                 _hubLifetimeManager.OnConnectedAsync(hubConnection).Wait();
+
+                _ = ConsumeAsync(connection.Application);
```

Comment:
Did you run the before and after with this?

Reply:
Yes.